### ISSUE TITLE
NOT WORKING.. SUDDENLY

### ISSUE THREAD

**Issue by Zamjirou** [Gunthersuper/steam-idle-bot#4]
HELP MEEEEEEE

**Comment by zeyfx** [comment]
yeah I did everything right too, but it doesn't work, all the content entered in Config.json has been confirmed and I deleted the old app created in Hiroku to avoid conflict

**Comment by Zamjirou** [comment]
only 1 main account works for me..

**Comment by batak1337** [comment]
error: failed to push some refs to 'https://git.heroku.com/polar-eyrie-91557.git' get this error after "git push heroku main"

**Comment by zeyfx** [comment]
> only 1 main account works for me..

Intentional. you can only have 1 main account and that's it
font: https://github.com/Gunthersuper/steam-idle-bot/issues/2


> error: failed to push some refs to 'https://git.heroku.com/polar-eyrie-91557.git' get this error after "git push heroku main"

go to Heroku > apps > turn off all apps through the individual settings of each one >

inside a new folder you will open PowerShell and type the following commands:
git config --global user.email YOUR EMAIL
git clone https://github.com/Gunthersuper/steam-idle-bot

A new folder "steam-idle-bot" will appeared in this directory. Go to this folder.

inside the "steam-idle-bot" folder you will open PowerShell and type the following commands:
heroku login
heroku create
git init
git add .
git commit -m 'commit'
git push heroku main        (note: yeah, its "main" and not "master")
heroku ps:scale web=0
heroku ps:scale bot=1

Hope I was helpfull
